ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/SandboxOffset.scala

```diff
@@ -10,10 +10,10 @@ object SandboxOffset {
   def numBytes = 8
 
   def toOffset(value: BigInt): Offset = {
-    Offset.fromBytes(value.toByteArray.reverse.padTo(numBytes, 0: Byte).reverse)
+    Offset.fromByteArray(value.toByteArray.reverse.padTo(numBytes, 0: Byte).reverse)
   }
 
   def fromOffset(offset: Offset): BigInt = {
-    BigInt(offset.toByteArray)
+    BigInt(offset.bytes.toByteArray)
```

Comment:
Calling `toByteArray` directly should work, right?

Reply:
fixed